### DISCUSSION TITLE
Ancient data (a.k.a. freezer) to a secondary drive?

### DISCUSSION THREAD

**Discussion by got3nks** [erigontech/erigon#5757] [Q&A] [answered]
Is it possible with Erigon to move ancient data (a.k.a. freezer) to a secondary drive, like you could do with _Go-Ethereum geth_?

**Comment by AskAlexSharov** [comment, marked as answer]
Done for blocks. See datadir/snapshots folder (—snapshots flag)
For history it will be done in Erigon3

**Reply by pxr64** [reply]
So can i move the snapshots folder to a HDD and symlink it?

**Reply by AskAlexSharov** [reply]
Yes: https://github.com/ledgerwatch/erigon#datadir-structure
